app/assets/stylesheets/application.css.scss.erb

```diff
@@ -381,3 +381,14 @@ td, th {
   text-align: center;
   padding: 10px;
 }
+
+input[type=text].modal-text {
+  height: 1.8rem;
+}
+
+.modal-radio[type="radio"]:not(:checked),
+.modal-radio[type="radio"]:checked {
+  left: 2px;
+  position: relative;
+  opacity: 1;
+}
```

Comment:
You can try to leverage more on SCSS syntax for better performance in .scss files